configs/detection.conf

```diff
@@ -1,6 +1,6 @@
 --obstacle_detection
 --nosimulator_obstacle_detection
---obstacle_detection_model_paths=dependencies/models/obstacle_detection/faster-rcnn/frozen_inference_graph.pb
+--obstacle_detection_model_paths=dependencies/models/obstacle_detection_tf2/faster-rcnn/
```

Comment:
Ideally, we would keep this as `obstacle_detection` and just remove the old models in the new version.